spec/30-processing-model.md

```diff
@@ -11,7 +11,7 @@ This processing model describes the behavior of a vendor that modifies and forwa
 If no traceparent header is received:
 
 1. The vendor checks an incoming request for a `traceparent` and a `tracestate` header.
-2. Because the `traceparent` header is not received, the vendor creates a new `trace-id` and `parent-id` that represents the current request.
+2. Because the `traceparent` header is not received, the vendor creates a new `trace-id` and `parent-id` that represents the current request. (Note: If the vendor does not sample this request and wants to communicate that sampling decision downstream via the `sampled` flag, it is acceptable to create a `trace-id` and `parent-id` that are not associated with any actual trace data. Not communicating the sampling decision downstream is equally acceptable.)
```

Comment:
I agree with this, but it may be worth rephrasing using normative words, e.g. "the vendor MAY decide to not sampling this request ... and MAY communicate ..."